skins/wmde19/src/error.ts

```diff
@@ -0,0 +1,22 @@
+import Vue from 'vue';
+import Error from './Error.vue';
+
+Vue.config.productionTip = false;
+
+const dataElement = document.getElementById( 'app' );
+let applicationVars: any = {}; // TODO Create interface
+let messages = {};
+
+if ( dataElement ) {
+	applicationVars = JSON.parse( dataElement.dataset.applicationVars || '{}' );
+	messages = JSON.parse( dataElement.dataset.applicationMessages || '{}' );
+}
+
+new Vue( {
+	render: h => h( Error, {
+		props: {
+			errorMessage: applicationVars.message,
```

Comment:
where is `message` coming from?

Reply:
It's assigned to the template in `InternalErrorHtmlPresenter` and `ErrorPageHtmlPresenter`.